brainstormforce/astra
# AST-1464 - Fixing  fatal error occurred after merging - Hide cart drawer (flyout) for mobile & tablet on checkout & cart page.

### Description
This PR is created for fixing fatal error in the `next-release` branch.
Optimized conditions for solving fatal error occurred when the woo-commerce plugin is deactivated.

### Screenshots
Fatal error - https://share.getcloudapp.com/DOumx8Nv

### Types of changes
Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
1. Checkout to `hide-cart-flyout-mobile` branch with was merged recently for [TASK](https://wp-astra.atlassian.net/browse/AST-1464)
2. Open cart or checkout page, flyout should be hidden as expected.
3. Disable the `Woo-commerce` plugin.
4. Refresh, You will encounter a Fetal error. (You can check debug.log)
5. Follow the same steps with `hide-cart-flyout-mobile-fix` branch, which is created to solve this fatal error.
6. Error should not reproduce now.

### Checklist:
